Data Loader Documentation

strategicalliances opened the pull request.

strategicalliances committed:
Update mendix-data-loader.md

Intermediate commit. Will continue later.

strategicalliances committed:
Update mendix-data-loader.md

Added info about using the target schema/db as a staging area. and some other minor changes.

strategicalliances committed:
Update mendix-data-loader.md

Added a limitation about filtering of the data.

strategicalliances committed:
Update mendix-data-loader.md

Removed part about ACCOUNTADMIN having to execute the sql in a worksheet (Prerequisites).

MarkvanMents commented:
Hi there,
@katarzyna-koltun-mx would normally pick this up, but she is off for a couple of days.
Can it wait until Monday (27th) or would you like someone else to deal with it this week?

strategicalliances committed:
Update mendix-data-loader.md

Changed the hyperlink reference to the Mendix Data Loader Snowflake marketplace listing.

katarzyna-koltun-mx committed:
Update mendix-data-loader.md

katarzyna-koltun-mx committed:
Update mendix-data-loader.md

katarzyna-koltun-mx committed:
Update mendix-data-loader.md

katarzyna-koltun-mx committed:
Merge branch 'development' into strategicalliances-patch-1
